coral-hive/src/main/java/com/linkedin/coral/transformers/CoralRelToSqlNodeConverter.java

```diff
@@ -390,6 +392,29 @@ public SqlNode toSql(RexProgram program, RexNode rex) {
         }
         return super.toSql(program, rex);
       }
+
+      @Override
+      public SqlNode field(int ordinal) {
```

Comment:
Please add the comment for this override with example and UT (not needed if it affects the existing UTs).